src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -46,6 +46,438 @@ public new abstract class Arm64 : ArmBase.Arm64
             /// </summary>
             public static Vector128<double> Add(Vector128<double> left, Vector128<double> right) { throw new PlatformNotSupportedException(); }
 
+            // /// <summary>
+            // /// uint64x1_t vcgez_f64 (float64x1_t a)
+            // ///   A64: FCMGE Dd, Dn, #0
+            // /// </summary>
+            // public static Vector64<ulong> CompareGreaterThanOrEqualZero(Vector64<double> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint16x4_t vcgez_s16 (int16x4_t a)
+            ///   A64: CMGE Vd, Vn, #0
+            /// </summary>
+            public static Vector64<ushort> CompareGreaterThanOrEqualZero(Vector64<short> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint32x2_t vcgez_s32 (int32x2_t a)
+            ///   A64: CMGE Vd, Vn, #0
+            /// </summary>
+            public static Vector64<uint> CompareGreaterThanOrEqualZero(Vector64<int> value) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// uint64x1_t vcgez_s64 (int64x1_t a)
+            // ///   A64: CMGE Dd, Dn, #0
+            // /// </summary>
+            // public static Vector64<ulong> CompareGreaterThanOrEqualZero(Vector64<long> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint8x8_t vcgez_s8 (int8x8_t a)
+            ///   A64: CMGE Vd, Vn, #0
+            /// </summary>
+            public static Vector64<byte> CompareGreaterThanOrEqualZero(Vector64<sbyte> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint32x2_t vcgez_f32 (float32x2_t a)
+            ///   A64: FCMGE Vd, Vn, #0
+            /// </summary>
+            public static Vector64<uint> CompareGreaterThanOrEqualZero(Vector64<float> value) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// uint64x1_t vcgez_u64 (uint64x1_t a)
+            // ///   A64: CMGE Dd, Dn, #0
+            // /// </summary>
+            // public static Vector64<ulong> CompareGreaterThanOrEqualZero(Vector64<ulong> value) { throw new PlatformNotSupportedException(); }
```

Comment:
It actually looks like `Vector64<long>/Vector64<ulong` are explicitly reserved today.